### PR TITLE
Updating Register Client Application Steps

### PR DESCRIPTION
**Pull request by dke-benjy**
to include the AIP UL Client GUID c00e9d32-3c8d-4a7d-832b-029040e7db99 which has been missing.

### CODE REVIEW

**Review by Josegto1 — APPROVED** [review]

**Review by Josegto1 — CHANGES_REQUESTED** [review]
Specific?

**Review by Josegto1 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/double-key-encryption.md [hunk 1]
```diff
@@ -526,6 +526,8 @@ To register the DKE service:
 
     4. Select **Save** at the top to save your changes.
 
+    5. Repeat the above steps, but this time define the Client ID as **c00e9d32-3c8d-4a7d-832b-029040e7db99**. This value is the Azure Information Protection Unified Labeling Client ID.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/double-key-encryption.md]:
```suggestion
    5. Repeat these steps, but this time, define the client ID as **c00e9d32-3c8d-4a7d-832b-029040e7db99**. This value is the Azure Information Protection unified labeling client ID. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@dke-benjy

**Comment by yogkumgit** [comment]
Gentle reminder @dke-benjy

**Pull request merged**